src/config/subgraph.ts

```diff
@@ -1,18 +1,35 @@
-import { ARBITRUM, AVALANCHE } from "./chains";
+import { ARBITRUM, AVALANCHE, ETHEREUM } from "./chains";
+import { isDevelopment } from "./env";
 
-export const SUBGRAPH_URLS = {
+const SUBGRAPH_URLS = {
   [ARBITRUM]: {
+    // stats: "https://api.thegraph.com/subgraphs/name/gdev8317/gmx-arbitrum-stats
     stats: "https://api.thegraph.com/subgraphs/name/gmx-io/gmx-stats",
     referrals: "https://api.thegraph.com/subgraphs/name/gmx-io/gmx-arbitrum-referrals",
     nissohVault: "https://api.thegraph.com/subgraphs/name/nissoh/gmx-vault",
   },
 
   [AVALANCHE]: {
+    // stats: "https://api.thegraph.com/subgraphs/name/gdev8317/gmx-avalanche-staging", // testing
     stats: "https://api.thegraph.com/subgraphs/name/gmx-io/gmx-avalanche-stats",
     referrals: "https://api.thegraph.com/subgraphs/name/gmx-io/gmx-avalanche-referrals",
   },
 
-  common: {
+  [ETHEREUM]: {
     chainLink: "https://api.thegraph.com/subgraphs/name/deividask/chainlink",
   },
 };
+
+export function getSubgraphUrl(chainId: number, subgraph: string) {
+  if (isDevelopment()) {
+    const localStorageKey = `subgraphUrl:${chainId}:${subgraph}`;
```

Comment:
let's store the key prefix in `config/localStorage`.

it might help to be aware of all stored values